### ISSUE TITLE
Upper time bound is ignored

### ISSUE THREAD

**Issue by abailly** [bbougon/git-stats#26]
**Describe the bug**

Running the following command 

```
gitflows-stats github $(cat github-token) input-output-hk hydra 2022-09-01 2023-05-31 --format html
```

yields this stats page

<img width="1206" alt="Screenshot 2023-06-16 at 09 34 27" src="https://github.com/bbougon/git-stats/assets/40221/a52b15f5-157d-4e05-93ef-26985bac2bfe">

I would have expected the period to end on May 31st. This is not only annoying for display but skews the stats and makes it impossible to extract evolution of average over time, eg. a sliding window of 3 months for example, in order to identify trends.

**Comment by bbougon** [comment]
Hello @abailly ,
Did you try to separate the 2 dates with a comma (`2022-09-01,2023-05-31` instead of `2022-09-01 2023-05-31`)?

if you run `gitflows-stats github --help` you will have the help displayed where it explains about the format. I'll should change the place where the format is detailed

**Comment by abailly-iohk** [comment]
🤦 
Perhaps the CLI arguments parser should be stricter and fail in this case?

**Comment by bbougon** [comment]
I like the idea. I'll check that 👍

**Comment by abailly-iohk** [comment]
So the displayed range is good:

<img width="1083" alt="Screenshot 2023-06-16 at 09 59 58" src="https://github.com/bbougon/git-stats/assets/79840582/fca60444-9024-4c2c-9860-3eb72c6a3b34">

but the graph still shows `Jun` 

<img width="689" alt="Screenshot 2023-06-16 at 10 00 06" src="https://github.com/bbougon/git-stats/assets/79840582/48be4ed9-bf9e-4caa-a2be-8fd1540cee6a">

**Comment by bbougon** [comment]
Ok, i'll check that ;)

**Comment by bbougon** [comment]
Issue is fixed on release 0.2.8
